Dockerfile

```diff
@@ -88,14 +89,14 @@ RUN apt -y update \
 #   pydicom: DICOM images
 #
 
-RUN pip --no-cache-dir install --upgrade pip setuptools wheel ipython
+RUN pip --no-cache-dir install --upgrade pip setuptools wheel pycocotools
 
 #
 # Install FiftyOne from source
 #
 
-COPY dist dist
-RUN pip --no-cache-dir install dist/*.whl && rm -rf dist
+# COPY dist dist
+# RUN pip --no-cache-dir install dist/*.whl && rm -rf dist
```

Comment:
This dockerfile originally is used to start `ipython` with `fiftyone` pre-installed.